test/intl402/DateTimeFormat/prototype/resolvedOptions/hourCycle-default.js

```diff
@@ -9,48 +9,38 @@ description: >
 info: |
   12.4.5 Intl.DateTimeFormat.prototype.resolvedOptions()
 
-  12.1.1 InitializeDateTimeFormat ( dateTimeFormat, locales, options )
-   29. If dateTimeFormat.[[Hour]] is not undefined, then
-     a. Let hcDefault be dataLocaleData.[[hourCycle]].
-     b. Let hc be dateTimeFormat.[[HourCycle]].
-     c. If hc is null, then
-        i. Set hc to hcDefault.
-     d. If hour12 is not undefined, then
-        i. If hour12 is true, then
-           1. If hcDefault is "h11" or "h23", then
-              a. Set hc to "h11".
-           2. Else,
-              a. Set hc to "h12".
-        ii. Else,
-           1. Assert: hour12 is false.
-           2. If hcDefault is "h11" or "h23", then
-              a. Set hc to "h23".
-           3. Else,
-              a. Set hc to "h24".
-     e. Set dateTimeFormat.[[HourCycle]] to hc.
-     
+  12.1.1 InitializeDateTimeFormat ( dateTimeFormat, locales, options)
```

Comment:
```suggestion
  11.1.2 CreateDateTimeFormat ( newTarget, locales, options, required, defaults )
```

Reply:
updated to address Ms2ger feedback, also added comment to hourCycle-dateStyle.js indicating what the tests test.